recording_server/src/meeting/meet/closeMeeting.ts

```
@@ -29,37 +29,31 @@ export async function closeMeeting(page: Page): Promise<void> {
                         break
                     }
                 } catch (e) {
-                    // Ignore errors and continue
+                    // Ignorer les erreurs et continuer
                     continue
                 }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)